### ISSUE TITLE
Read and edit more

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#335]
```
What new feature do you want?
Add option to allow view and edit post text that is aftrer <!--more--> splitter 
(in Blogger) with ScribeFire.
```

-----
Original issue reported on code.google.com by `arvis.la...@gmail.com` on 14 May 2011 at 5:39

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `cfinke` on 3 Nov 2011 at 3:19
* Added labels: **Type-Enhancement**

**Comment by GoogleCodeExporter** [comment]
```
Issue 362 has been merged into this issue.
```

-----
Original comment by `cfinke` on 3 Nov 2011 at 3:25

**Comment by GoogleCodeExporter** [comment]
```
It doesn't work for old posts.
```

-----
Original comment by `muskasiy...@ymail.com` on 14 Dec 2011 at 12:51

**Comment by GoogleCodeExporter** [comment]
```
Fixed for blogger blogspot in version 4.3.1
```

-----
Original comment by `scribefi...@gmail.com` on 14 Apr 2014 at 7:01
* Changed state: **Fixed**
